docs(grid, treelist):added section for searchbox in the state of the … (#613)
telerik/blazor-docs: components/grid/filter/searchbox.md

![grid search box](images/search-box-overview.gif)

## Filter From Code

> You can set the Grid filters from your code through the component [state]({%slug grid-state%}).

Reviewer: ```suggestion
You can set the Grid filters programmatically through the component [state]({%slug grid-state%}).
```
